spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkCopyOnWriteScan.java

```diff
@@ -123,8 +127,17 @@ public void filter(Filter[] filters) {
               tasks().stream()
                   .filter(file -> fileLocations.contains(file.file().path().toString()))
                   .collect(Collectors.toList());
+
+          LOG.info(
```

Comment:
Unlike `SparkBatchQueryScan`, this is a special type of filtering where we know it is an IN filter with file locations. If we sanitize the expression, logging it will be useless. I added the number of files to the log, though.

Let me know what you think.

Reply:
Oh I see, because most files have date/timestamp in name, and those get sanitized.  Makes sense to me.